### DISCUSSION TITLE
Disclosure Animation

### DISCUSSION THREAD

**Discussion by alekseykurylev** [ariakit/ariakit#3477] [Questions] [answered]
Hi, will animation be added for the Disclosure component? How it was done in version 0.4 https://newsletter.ariakit.org/p/animation-ariakit-plus-and-more

**Comment by georgekaran** [comment, marked as answer]
Hi @alekseykurylev, 

You can now use animations on the `Disclosure` component by using the `data-enter` attribute. Please check out my example here: https://stackblitz.com/edit/ttry2s?file=disclosure/style.css,disclosure/index.tsx&theme=dark. 

If you have any questions, feel free to ask!
